flask_table/table.py

```diff
@@ -163,4 +168,18 @@ def create_table(name=str('_Table'), base=Table):
         # Then assume that what we have is a single class, so make it
         # into a 1-tuple.
         base = (base,)
-    return TableMeta(name, base, {})
+
+    attr_dict = {}
+    done = False
+    for base_class in base:
+        if not done:
+            for option in options:
+                if option in base_class.__dict__:
+                    attr_dict[option] = options[option]
+                    if len(attr_dict) == len(options):
+                        done = True
+                        break
+        else:
+            break
```

Comment:
If not there are other ways ? the second `break` is to quit while all attributes checked when we have several superclasses table, for instance our table is mainly single inheritance from `Table`.